src/settings/submit-rageshake.ts

```diff
@@ -9,10 +9,10 @@ import { type ComponentProps, useCallback, useEffect, useState } from "react";
 import { logger } from "matrix-js-sdk/src/logger";
 import {
   ClientEvent,
-  type Crypto,
   type MatrixClient,
   type MatrixEvent,
 } from "matrix-js-sdk/src/matrix";
+import { type CryptoApi } from "matrix-js-sdk/src/crypto-api";
```

Comment:
I'm nor sure why these are in this PR? 🤷

Reply:
Because we updated the js-sdk. And they recently deprecated non rust crypto and did some breaking changes.